publishing/s3publisher.py

```diff
@@ -195,3 +205,24 @@ def publish_to_s3(directory, base_url, site_prefix, bucket, cache_control,
 
             delta = datetime.now() - start_time
             LOGGER.info(f'... done in {delta.total_seconds():.2f}s')
+
+
+def update_admin_config(filename, base_url, owner, repository, branch,
+                        auth_base_url, auth_endpoint, bucket_type):
+    config = None
+    try:
+        with open(filename) as f:
+            config = yaml.safe_load(f)
+            if config:
+                if 'backend' in config:
```

Comment:
What is the effect of overwriting `backend` ONLY if it exists? This means that if it doesn't exist, we won't configure it and Netlify CMS won't work, I don't think that it is the intention. I think we should provide this configuration regardless of whether `backend` exists already, this way the user never has to configure at all if they don't want it when developing locally.

Reply:
okay ... yeah ... in the commit history you can see i had removed this requirement but then re-introed as it bacame a little tricky ... i will revisit

Reply:
hmmmm, you have a point about the 404 stuff, its already there. I suppose its fine. It does seem odd to me that we have to provide the information again when we already to it in the build, and thinking about this I'm confused as to why the federalist app is sending a message via SQS to "publish" the app, wouldn't that just happen after the build?

Reply:
sqs puts a build in the queue ... a single build process is composed of multiple tasks (clone, federalist-script, build, and publish)

Reply:
@davemcorwin - backend is no longer required ... i think if we want to break up s3publisher functionality that is another issue